api/index.ts

```diff
@@ -0,0 +1,62 @@
+export const BASE_API_URL =
+  process.env.NODE_ENV !== "production"
+    ? "http://localhost:3000"
+    : process.env.BASE_API_URL;
+
+export const ENDPOINTS = {
+  HOSPITALS: `${BASE_API_URL}/hospitals`,
+  USERS: `${BASE_API_URL}/users`,
+  LOGIN: `${BASE_API_URL}/login`,
+};
+
+export async function getHospitals() {
+  return fetch(ENDPOINTS.HOSPITALS, {
+    method: "GET",
+    headers: await getRequestsHeaders(),
```

Comment:
As we have commented, we should remove **await** since getRequestHeaders should not be an async function